hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,154 @@
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_aes_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_hmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_kmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_otbn_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_main_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_io_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_io_div2_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_io_div4_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_usb_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_io_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "3003057152" "(!rst_ni)" (4) "(!rst_ni) 0,0,0,0"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_io_div2_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "3003057152" "(!rst_ni)" (4) "(!rst_ni) 0,0,0,0"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_io_div4_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
```

Comment:
The u_timeout_ref_to_clk seem to be due to a missing else in the flop at https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/prim/rtl/prim_clock_timeout.sv;drc=df6bbf5f67659efa109d90a1d730a68d35a5a590;l=40

Reply:
but doesn't the missing `else` just mean the count remains the same?

Reply:
Yes it does, but coverage complains we never visit the "else". It is unreachable probably because !en_i is already dealt with in line 34.